src/gamepad.rs

```diff
@@ -33,7 +33,8 @@ impl<S: StickIdType> Plugin for GamepadMappingPlugin<S> {
 }
 
 /// HACK: chosen at random, we're betting on no collisions with gilrs gamepads
-const TOUCH_GAMEPAD_ID: usize = 6101115544193436746;
+/// updated too work on 32bit platforms, NOT u32::MAX, a bit less.
```

Comment:
Oops, didn't see this yesterday, but comments about what changed fit better as git commit messages. I updated this in main now.